### PR TITLE
updated input fields to select options and validations accordingly

### PR DESCRIPTION
**Pull request by Ilze-R**

### CODE REVIEW

File: src/app/monthly-calc/monthly-calc.component.ts [hunk 1]
```diff
@@ -10,29 +10,12 @@ const fb = new FormBuilder().nonNullable;
...
-          Validators.required,
-          Validators.max(this.kidsAmount),
-          Validators.pattern('[0-9]*'),
-        ],
-      ],
+      applicants: ['', Validators.required],
+      amountOfKids: ['', Validators.required],
       income: ['', [Validators.required, Validators.pattern('[0-9]*')]],
-      obligations: ['', [Validators.required, Validators.pattern('[0-9]*')]],
+      obligations: ['', Validators.pattern('[0-9]*')],
```
Comment (kyskiz) [inline src/app/monthly-calc/monthly-calc.component.ts:18]: Is obligations used in template?

Comment (Ilze-R) [reply src/app/monthly-calc/monthly-calc.component.ts:18]: <mat-form-field class="input-wrap">
        <mat-label>Obligations</mat-label>
        <input matInput formControlName="obligations" />
        <mat-error
          *ngIf="
            obligations && obligations.errors && obligations.errors['pattern']
          "
        >
          Please enter a number
        </mat-error>
        <mat-icon
          matSuffix
          matTooltip="Represent any outstanding debts or regular payments that a party must make."
          [matTooltipPosition]="'right'"
          >help_outline</mat-icon
        >
      </mat-form-field>

Comment (Ilze-R) [reply src/app/monthly-calc/monthly-calc.component.ts:18]: ` <mat-form-field class="input-wrap">
        <mat-label>Obligations</mat-label>
        <input matInput formControlName="obligations" />
        <mat-error
          *ngIf="
            obligations && obligations.errors && obligations.errors['pattern']
          "
        >
          Please enter a number
        </mat-error>
        <mat-icon
          matSuffix
          matTooltip="Represent any outstanding debts or regular payments that a party must make."
          [matTooltipPosition]="'right'"
          >help_outline</mat-icon
        >
      </mat-form-field>`

Comment (Ilze-R) [reply src/app/monthly-calc/monthly-calc.component.ts:18]: I use line 18 to validate the input field "obligations" - if the inserted value is NOT string

Comment (Ilze-R) [reply src/app/monthly-calc/monthly-calc.component.ts:18]: In the code from branch month-validations

**Review by swoogie — DISMISSED** [review]
lgtm

File: src/app/monthly-calc/monthly-calc.component.ts (hunk 1 above)
Comment (swoogie) [reply src/app/monthly-calc/monthly-calc.component.ts:18]: Obligations is used in lines 51-59 in template

**Review by kyskiz — APPROVED** [review]

**Pull request merged**